frontend/packages/topology/src/components/side-bar/TopologySideBar.tsx

```diff
@@ -30,14 +30,16 @@ const TopologySideBar: React.FC<TopologySideBarProps> = ({ children, onClose })
       onResize={handleResizeCallback}
     >
       <PFTopologySideBar resizable className="pf-topology-side-bar-resizable">
-        <div className="co-sidebar-dismiss clearfix">
-          <CloseButton
-            onClick={onClose}
-            dataTestID="sidebar-close-button"
-            additionalClassName="co-close-button--float-right co-sidebar-dismiss__close-button"
-          />
+        <div className="pf-topology-side-bar__body">
```

Comment:
Why additional div here? can't we add directly to the parent ?
eg: <PFTopologySideBar resizable className="pf-topology-side-bar-resizable pf-topology-side-bar__body">

Reply:
Modified with above changes.

Reply:
@lokanandaprabhu @karthikjeeyar Looks like the body was required for a reason. With this extra div:

![extra-body-div](https://user-images.githubusercontent.com/139310/164993294-95ce44c9-fcd8-4156-913f-763ca2c5d066.gif)

The current solution breaks the scrolling (the sidebar scrolls the topology content area as well):

![currently-broken](https://user-images.githubusercontent.com/139310/164993324-d76747fc-e12d-49eb-b334-fe200e320dac.gif)

@lokanandaprabhu Can you please open another PR to fix this. I noticed this while testing the backport #11377. For the backport we need a manually cherry-pick of this and the new PR for 4.10. Let me know if you have any questions. :)